open_spiel/scripts/python_extra_deps.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Have you considered specifying min versions rather than equality? This should make the optional dependencies easier to use.

Reply:
I guess the reasoning in requirements.txt still applies? Maybe it would appropriate to move it here?

Reply:
I have changed requirements.txt to be lower bounds on the version. These are now entirely optional in the sense that if the user wants to use an algorithm or tool that depends on them, it is tgeir responsibility to install it separately.  These specific versions are never installed except on github actions CI for testing purposes, do they exist as recommendations to the user (not requirements)

Reply:
Makes sense! Maybe exactly this comment could be added to this file.

Reply:
Yeah agree, this conversation made me realize this has to be better documented. Will do. Thanks!

Reply:
I probably will rephrase "single source of truth" because that feels too strong regardless.